app/presenters/content_item/withdrawable.rb

```diff
@@ -9,11 +9,13 @@ def page_title
     end
 
     def withdrawal_notice_component
+      lang = I18n.locale.to_s == "en" ? false : "en"
```

Comment:
One little code nitpick. As this variable is only used once, could this ternary operator be moved to the `lang` attribute below instead of being passed via this variable?

Reply:
Good spot, updated.